### ISSUE TITLE
Can't assign a color to a submodule tab

### ISSUE THREAD

**Issue by Gama11** [ForkIssues/TrackerWin#1143]
The "Ability to assign color tags to repository tabs" feature from the latest release doesn't seem to work for submodule tabs. In fact, it's missing some of the other options too (rename, workspaces), it only has close actions:

![image](https://user-images.githubusercontent.com/2620907/114302311-d0abf880-9ac8-11eb-8a7d-32774bc1830e.png)

**Comment by DanPristupov** [comment]
This is a limitation which currently exists in Fork. You can't rename or assign a color to a submodule. Submodule tabs are also not restored after restart. This won't be changed in the near future.

**Comment by Gama11** [comment]
>Submodule tabs are also not restored after restart.

They are, actually, though I remember that was a limitation a while ago. :) From the 1.43 release notes:

```
Improved - Keep submodule tabs open after restart
```

**Comment by Stanzilla** [comment]
Yeah just noticed the same. Actually love the fact that submodule tabs are a thing in Fork, in other clients, like Tower, they open in the same tab which is super annoying.

**Comment by drB-sii** [comment]
Well I found a workaround (aka dirty hack) to actually do this: You can add the submodules as full repository entries in the settings.json file in the fork installation folder. 

**But beware that this can destroy your settings** if you do any mistakes. If this happens Fork reset the settings.json file on startup and resets all prior settings.
For instance: I had some gaps in the IDs and tried to fill the numbers with my submodules but for one submodule it didn't work out of the box. Maybe the submodules need higher numbers than their parent repo.

The GitDirectoryPath property should be something like "<parent git folder>\\.git\\modules\\<submodule folder>"
